routes/apis/users.js

```diff
@@ -23,9 +23,7 @@ router.post(
     ],
     async (req, res) => {
         const errors = validationResult(req);
-
-        console.log('Test');
-        console.log(req.query);
+        
         // If there are any errors found by validationResult in user's HTTP request, we'll return an HTTP status code 400
         // with an array of errors found
```

Comment:
Is there any situation where we need to account for 404 error? Disconnection from server?

Reply:
It's really up to us how to handle our errors. If you look at the try-catch statement I wrote out, I use a Error 500 instead of a 404 if I can't make a connection with the server.